src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/settings/SettingsPluginComponent.kt

```diff
@@ -18,10 +19,15 @@ class SettingsPluginComponent {
     var panel: JPanel? = null
 
     // Plugin description
-    private val pluginDiscription = JLabel(
-        "<html><body>TestGenie is an external graphical IntelliJ plugin that integrates EvoSuite into the IDE." +
-            " EvoSuite is a tool that automatically generates test cases with assertions for classes written in Java code." +
-            " TestGenie makes this much easier, as it provides an intuitive modern interface for EvoSuite – so, no more CLI."
+    private val pluginDescription = JLabel(
```

Comment:
why the `</br>`? Before that, the text stretched and shrunk with the window, dynamically. Now its kind of static.

Reply:
I guess we need to discuss the design choice. Wonder which one is better 🤔